src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +47,44 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void findStationNames() {
+        //given
+        List<Map<String, String>> stations = new ArrayList<>();
+        stations.add(Map.of("name", "부평구청역"));
+        stations.add(Map.of("name", "가산디지털단지역"));
+
+        stations.forEach(StationCommonApi::createStation);
+
+        // when
+        List<String> stationNames = StationCommonApi.findAllStations()
+            .jsonPath().getList("name", String.class);
+
+        //then
+        assertThat(stationNames).contains("부평구청역", "가산디지털단지역");
```

Comment:
```suggestion
        assertThat(stationNames).containsExactly("부평구청역", "가산디지털단지역");
```

정확한 비교를 위하여 `containsExactly`를 사용해 볼까요?